src/NMEAGPS.cpp

```diff
@@ -445,7 +447,7 @@ static const char * const std_nmea[] __PROGMEM =
 const NMEAGPS::msg_table_t NMEAGPS::nmea_msg_table __PROGMEM =
   {
     NMEAGPS::NMEA_FIRST_MSG,
-    (const msg_table_t *) NULL,
+    (const msg_table_t *) 0x00,
```

Comment:
Are you saying `NULL` is not defined on your platform?  Wow.

Please try `nullptr` instead.  This is a newer language feature intended to mean "a pointer to nothing", not just a `#define` to zero.